02-oop-inheritance/keypoints/2_7-12/script.js

```diff
@@ -0,0 +1,209 @@
+/**
+ * Created by MaximilianoDaniel on 17/09/2015.
+ */
+
+var inheritPrototype = function (childClass, parentClass){
+
+    var copyOfParent = Object.create(parentClass.prototype);
+
+    //Is necessary to point to the childClass constructor
+    copyOfParent.constructor = childClass;
+    childClass.prototype = copyOfParent;
+};
+
+function augment( receivingClass, givingClass ) {
+
+    // only provide certain methods
+    // the index starts at two because the first and second arguments are receivingClass and givingClass
+    if ( arguments[2] ) {
```

Comment:
@Tergopol  well ... I understand the next, the first argument is receivingClass, the second is givinClass. So if you need augment some method it's the third argument in the function. it's ok?